recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "tomlkit" %}
-{% set version = "0.9.2" %}
+{% set version = "0.10.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/tomlkit-{{ version }}.tar.gz
-  sha256: ebd982d61446af95a1e082b103e250cb9e6d152eae2581d4a07d31a70b34ab0f
+  sha256: 30d54c0b914e595f3d10a87888599eab5321a2a69abc773bbefff51599b72db6
 
 build:
   number: 0
```

Comment:
We do not build `noarch: python` anymore so you can remove it.  But add `skip: True # [py<36]`, see https://github.com/sdispater/tomlkit/blob/e15bb79401517ca28013454662741ca9b0646fab/pyproject.toml#L21.
If so we need to use a plain `python` without pinning in `run`

Reply:
The build backend is **poetry**, so `setuptools` isn't needed, see https://github.com/sdispater/tomlkit/blob/e15bb79401517ca28013454662741ca9b0646fab/pyproject.toml#L61